internal/cli/cli.go

```diff
@@ -190,6 +190,10 @@ func (c *cli) getTenant() (tenant, error) {
 		return tenant{}, fmt.Errorf("Unable to find tenant: %s", c.tenant)
 	}
 
+	if t.Apps == nil {
+		t.Apps = map[string]app{}
+	}
```

Comment:
bugfix: without this we had nil panics when trying to assign to the apps map on successful login.

Reply:
Good catch!